.github/workflows/ci.yml

```diff
@@ -4,13 +4,19 @@ jobs:
   test:
     strategy:
       matrix:
+        java: ['1.8', '11.0.5']
```

Comment:
Why force specific minor update?

Reply:
Yeah, you're right.
I introduced that in asciidoctorj as I had problems with gradle failing due to Out Of Metaspace  errors and during experimentation I figured that I didn't get the latest Java version, but 11.0.1 at that time. No idea if it is the current version now.

However I just changed it to only use '11'